jsonschema-core.xml

```diff
@@ -2459,6 +2443,12 @@
                             array element even after the first match has been found.  This
                             is to ensure that all possible annotations are collected.
                         </t>
+                        <t>
+                            This keyword produces an annotation value which is an array of
+                            the indexes to which this keyword successfully applied its subschema,
+                            in ascending order.  The value MAY be a boolean true if the subschema
+                            was successfully applied to every index of the instance.
```

Comment:
I'm not happy with the phrasing `successfully applied`. I feel it's ambigious and it's the first time we've used it in this document. This is the only two instances of it.

I'd be happy with `validates successfully when applied` or similar.

```suggestion
                            This keyword produces an annotation value which is an array of
                            the indexes to which this keyword validates successfully when applying 
                            its subschema, in ascending order. The value MAY be a boolean true if the 
                            subschema validated successfully when applied to every index of the instance.
```